PWM.py

```diff
@@ -0,0 +1,18 @@
+#code works along side GPIO.py
+#the init in GPIO.py must be called before any of these functions are called
```

Comment:
can the `init()` function be called more than once? if so, it'd be better to call it in the `Signal` class constructor.

Reply:
I mean I guess, it just felt kind of redundant to do the same thing twice